app/components/channel_loader/channel_loader.js

```diff
@@ -150,8 +47,8 @@ export default class ChannelLoader extends PureComponent {
         const style = getStyleSheet(theme);
 
         return (
-            <View style={[style.container, {width: deviceWidth}]}>
-                {Array(20).fill().map((item, index) => this.buildSections({key: index, style, top: index === 0, bg: theme.centerChannelBg}))}
+            <View style={[style.container]}>
+                {Array(6).fill().map((item, index) => this.buildSections({key: index, style, bg: theme.centerChannelBg, color: theme.centerChannelColor}))}
```

Comment:
Can you pass a prop `backgroundColor`, if this.props.backgroundColor exists use that instead of theme.centerChannelBg.

That way we do not depend on the theme colors since it may not be ready with the Entry component

Reply:
will do, at some point later today